com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/JdtSourceLookUpProvider.java

```diff
@@ -150,7 +150,10 @@ public boolean supportsRealtimeBreakpointVerification() {
                 ValidBreakpointLocationLocator locator = new ValidBreakpointLocationLocator(astUnit, lines[i], true, true);
                 astUnit.accept(locator);
                 // When the final valid line location is same as the original line, that represents it's a valid breakpoint.
-                if (lines[i] == locator.getLineLocation()) {
+                // Add location type check to avoid breakpoint on method/field which will never be hit in current implementation.
+                if (locator.getLocationType() == ValidBreakpointLocationLocator.LOCATION_LINE) {
```

Comment:
Update the javadoc for `getFullyQualifiedName()` because the behavior is changed.

Reply:
updated

Reply:
In line 143, we have a todo item that indicates future support of moving breakpoint location to a valid one. Please remove the comments if you just implemented this.